src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -4249,6 +4249,14 @@ compiler.warn.declared.using.preview=\
 compiler.warn.attempt.to.synchronize.on.instance.of.value.based.class=\
     attempt to synchronize on an instance of a value-based class
 
+# lint: identity
+compiler.warn.attempt.to.synchronize.on.instance.of.value.based.class2=\
```

Comment:
How is this different from the message with no 2 suffix? Is there a reason why one localization entry is required for one type of lint?

Reply:
the requirement is for the identity lint warning to be an alias of the synchronization warning